internal/constants/constants.go

```diff
@@ -44,8 +44,7 @@ const (
 	TrustedCABundleHashName    = "logging.openshift.io/hash"
 	SecretHashPrefix           = "logging.openshift.io/"
 	KibanaTrustedCAName        = "kibana-trusted-ca-bundle"
-	// internal elasticsearch FQDN to prevent to connect to the global proxy
```

Comment:
Is this because when the global proxy is enabled there is no way to filter traffic via NO_PROXY? meaning we will always be writing through the global proxy

Reply:
Would "NO_PROXY=cluster.local" work?

Reply:
I don't but I will defer to @alanconway here since he gave it a :lgtm